docs/content/latest/develop/gitdev/gitpod.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Gitpod
+linkTitle: Gitpod
+description: Ready to code Gitpod integrated dev environment
+menu:
+  latest:
+    identifier: gitpod
+    parent: gitdev
+    weight: 592
+isTocNested: true
+showAsideToc: true
+---
+
+Use the [Gitpod](https://www.gitpod.io) workspaces to provision an instant development environment with a pre-configured YugabyteDB.
+
+Gitpod is a configurable ready to code cloud development environment accessible via a browser. A Gitpod workspace includes everything developers need to develop for a specific repository, including the `vscode` or `theia` editing experience, common languages, tools, and utilities. Instantly it sets up a cloud-hosted, containerized, and customizable vscode/theia environment.
```

Comment:
Theia is gone as of April: https://www.theregister.com/2021/04/08/gitpod_talks_up_importance_of/

```suggestion
Gitpod is a configurable ready-to-code cloud development environment accessible via a browser. A Gitpod workspace includes everything you need to develop for a specific repository, including the Visual Studio Code editing experience, common languages, tools, and utilities. Instantly it sets up a cloud-hosted, containerized, and customizable editing environment.
```